app/components/chat/chat-input.tsx

```diff
@@ -155,6 +155,7 @@ export default function ChatInput(props: ChatInputProps) {
     if (isURL(input)) {
       setTemporaryURLInput(input);
     }
+    setUserInput("");
```

Comment:
@jwandekoken this looks like a great catch, thanks! Currently it's only cleared after the response. I guess this call (clearing it after the response) is still there and has to be removed then in this PR too.

Reply:
Thanks for pointing that out! I removed this reset after the response, see if that is OK now.